pocs/utils/logger.py

```diff
@@ -66,32 +72,40 @@ def get_root_logger(profile='panoptes', log_config=None):
             log_config['formatters'][name].setdefault('()', _UTCFormatter)
         log_fname_datetime = datetime.datetime.utcnow().strftime('%Y%m%dT%H%M%SZ')
     else:
-        log_fname_datetime = datetime.datetime.now().strftime('%Y%m%dT%H%M%SZ')
+        log_fname_datetime = datetime.datetime.now().strftime('%Y%m%dT%H%M%S')
 
     # Setup log file names
     invoked_script = os.path.basename(sys.argv[0])
-    log_dir = '{}/logs'.format(os.getenv('PANDIR', gettempdir()))
-    log_fname = '{}-{}-{}'.format(invoked_script, os.getpid(), log_fname_datetime)
-    log_symlink = '{}/{}.log'.format(log_dir, invoked_script)
+    log_dir = os.getenv('PANLOG', '')
+    if not log_dir:
+        log_dir = os.path.join(os.getenv('PANDIR', gettempdir()), 'logs')
```

Comment:
Why the temp dir here? Seems like you could just put it in `$PANDIR/logs`

Reply:
That was there before... It handles the case where PANDIR has not been set (i.e. gettempdir is providing the default value in case PANDIR is unset).